spec/spec_helper.rb

```diff
@@ -1,21 +1,29 @@
+# frozen_string_literal: true
+
 SPEC_DIR = File.dirname(__FILE__)
-FIXTURES_DIR = File.join(SPEC_DIR, "fixtures")
+FIXTURES_DIR = File.join(SPEC_DIR, 'fixtures')
 
-$LOAD_PATH.unshift File.join(SPEC_DIR, "..", "lib")
+$LOAD_PATH.unshift File.join(SPEC_DIR, '..', 'lib')
 $LOAD_PATH.unshift SPEC_DIR
 
 require 'dotenv'
 Dotenv.load
 
-require 'wcc/api'
+require 'webmock/rspec'
+Dir['./spec/support/**/*.rb'].sort.each { |f| require f }
 
-Dir[File.join(SPEC_DIR, "support", "*.rb")].each do |support_file|
-  require "support/#{File.basename(support_file, ".rb")}"
-end
+require 'wcc/api'
 
 RSpec.configure do |config|
-  config.run_all_when_everything_filtered = true
-  config.filter_run :focus
+  config.expect_with :rspec do |expectations|
+    expectations.include_chain_clauses_in_custom_matcher_descriptions = true
+  end
+
+  config.mock_with :rspec do |mocks|
+    mocks.verify_partial_doubles = true
+  end
+
+  config.shared_context_metadata_behavior = :apply_to_host_groups
 
-  config.order = 'random'
```

Comment:
This should be added to either the `.rspec` or at least the circleci build - want to be able to catch test cleanup errors.

Reply:
the circle config does specify `rand` in the `rspec` command